fl4health/clients/nnunet_client.py

```diff
@@ -259,16 +264,20 @@ def get_lr_scheduler(self, config: Config) -> _LRScheduler:
             )
 
         # Determine total number of steps throughout all FL rounds
-        local_epochs, local_steps, *_ = self.process_config(config)
+        local_epochs, local_steps, _, _ = self.process_config(config)
         if local_steps is not None:
-            self.steps_per_round = local_steps
+            steps_per_round = local_steps
+            total_steps = int(config["n_server_rounds"]) * steps_per_round
         elif local_epochs is not None:
-            self.steps_per_round = local_epochs * len(self.train_loader)
-        self.max_steps = int(config["n_server_rounds"]) * self.steps_per_round
+            steps_per_round = local_epochs * len(self.train_loader)
+            total_steps = int(config["n_server_rounds"]) * steps_per_round
+        else:
+            raise ValueError("One of local steps and local epochs must be None")
```

Comment:
I think you might mean 'One of local_steps and local_epochs must specified'. This exception is only reached when they are both None.